models/user.js

```diff
@@ -1,30 +1,32 @@
 const database = require('../utils/database');
 const bcrypt = require('bcrypt');
 
-let convertToUserObject = function(DBUser) 
-{
-	return 	{
-				id: 			DBUser.US_PK,
-				username: 		DBUser.US_Username 		|| null,
-				password: 		DBUser.US_Password 		|| null,
-				email: 			DBUser.US_Email 		|| null,
-				firstName: 		DBUser.US_FirstName 	|| null,
-				lastName: 		DBUser.US_LastName 		|| null,
-				phoneNumber: 	DBUser.US_PhoneNumber 	|| null,
-				birthDate: 		DBUser.US_BirthDate 	|| null,
-				joinDate: 		DBUser.US_JoinDate 		|| null
-			};
-}
+let convertToUserObject = function (DBUser) {
+    return {
+        id: DBUser.US_PK,
+        username: DBUser.US_Username || null,
+        password: DBUser.US_Password || null,
+        email: DBUser.US_Email || null,
+        firstName: DBUser.US_FirstName || null,
+        lastName: DBUser.US_LastName || null,
+        phoneNumber: DBUser.US_PhoneNumber || null,
+        birthDate: DBUser.US_BirthDate || null,
+        joinDate: DBUser.US_JoinDate || null
+    };
+};
 
 /**
- * Retrieves the user from the database by username. 
+ * Retrieves the user from the database by username.
  * @param {string} username - The username of the user.
  * @param {userObject} callback - found() and notFound() expected
  */
-exports.getUser = function(username, callback = {found: (user) => {}, notFound: () => {}}) 
-{
-	var db = database.connectDatabase();
-	var query = `
+exports.getUser = function (username, callback = {
```

Comment:
leave as it was pls
or alternatively like this:
found: (user) => { },
notFound: () => { }